views/route.html

```diff
@@ -0,0 +1,33 @@
+<div>
+  <kubernetes-object-describe-header resource="resource" kind="kind"></kubernetes-object-describe-header>
+  <dl class="dl-horizontal">
+    <dt>Name</dt>
+    <dd>{{resource.metadata.name}}</dd>
+    <dt>Namespace</dt>
+    <dd>{{resource.metadata.namespace}}</dd>
+    <dt>Created</dt>
+    <dd>{{resource.metadata.creationTimestamp}}</dd>
+    <dt>Host</dt>
+    <dd>{{resource.host}}</dd>
+    <dt>Path</dt>
+    <dd>{{resource.path || 'None'}}</dd>
+    <dt>Service Name</dt>
+    <dd>{{resource.serviceName}}</dd>
+  </dl>
+  <div ng-if="resource.tls">
```

Comment:
add `&& resource.tls.termination`... from API docs (https://github.com/openshift/origin/blob/master/pkg/route/api/v1beta3/types.go#L75-L92): "Termination indicates termination type.  If termination type is not set, any termination config will be ignored"